internal/pally/metric.go

```diff
@@ -66,6 +68,23 @@ type GaugeVector interface {
 	MustGet(...string) Gauge
 }
 
+// Latencies approximates a latency distribution with a histogram.
+//
+// Latencies are exported to both Prometheus and Tally using their native
+// histogram types.
+type Latencies interface {
+	Observe(time.Duration)
```

Comment:
I think that this should be captured by the type; we're asking for a time.Duration, which should clearly signal that there's no multiplication or division required.